### PR TITLE
Update index_de.md

### PR DESCRIPTION
**Pull request by TheChymera**
Style improvements

### CODE REVIEW

File: index_de.md [hunk 1]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
-dabei zuschauen wie dies bereits in organisierter Form anderen
-Aktivisten der Freien Software-Bewegung passiert ist.
-Wir werden dieses Mal nicht tatenlos dabei zusehen, wie ein Symbol 
-dieser Gemeinschaft angegriffen wird.
+Richard M. Stallman, häufig bekannt unter dem Kürzel RMS, war seit
+Jahrzehnten eine treibende Kraft in der Freien Software-Bewegung,
+mit Beiträgen wie dem GNU Betriebssystem und Emacs.
+
+In jüngster Zeit wurden eine reihe verwerflicher Online-Angriffe
+bewerkstelligt, mit dem Ziel ihn aus dem Vorstand der FSF zu entfernen,
```
Comment (louis77) [inline index_de.md:12]: Better: "Online-Angriffe lanciert"
Meaning: "gezielt in die Öffentlichkeit gelangen lassen"

File: index_de.md [hunk 2]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
-
-Unabhängig davon: Stallman's Meinungen, für die er verfolgt wird, 
-sind nicht relevant hinsichtlich seiner Fähigkeit, eine Gemeinschaft wie 
-die FSF zu führen. 
-Darüber hinaus hat er genauso Anspruch auf seine eigenen Meinungen 
-wie jeder andere auch. Mitglieder und Unterstützer 
-müssen nicht zwingend mit seinen Ansichten übereinstimmen, 
-sollten aber sein Recht auf Gedanken- und Redefreiheit respektieren.
+seine Mitglieder fair und unvoreingenommen zu behandeln und sollte
+politischem Druck von außen nicht nachgeben. Wir fordern die FSF auf,
```
Comment (louis77) [inline index_de.md:21]: is "political pressure" and "social pressure" the same? 
Evtl. "Gruppendruck" ?

File: index_de.md [hunk 3]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+die Argumente gegen RMS objektiv zu betrachten und die eigentliche
+Bedeutung seiner Worte und Taten — im lichte der persönlicher Freiheit,
```
Comment (louis77) [inline index_de.md:23]: very nice! little typo: "im **L**ichte der persönliche*n*"

File: index_de.md [hunk 4]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+die ja den Hauptwert der Freien Software Bewegung darstellt — zu
+verstehen.
+
+In der Vergangenheit hat RMS seine Ansichten in einer Weise zum
+Ausdruck gebracht, die durchaus verärgerten könnte. Er konzentriert
```
Comment (louis77) [inline index_de.md:28]: Original "... in ways that upset many people." has a different meaning than "durchaus verärgerten könnte".
We shouldn't downplay by using subjunctives.

File: index_de.md [hunk 5]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+verstehen.
+
+In der Vergangenheit hat RMS seine Ansichten in einer Weise zum
+Ausdruck gebracht, die durchaus verärgerten könnte. Er konzentriert 
+sich in der Regel mehr auf philosophische Hintergründe sowie auf die Suche
+nach Wahrheit und pflegt sprachlichen Purismus. Dabei räumt er den
+Gefühlen der Menschen in seinen Kommentaren oft wenig Raum ein. 
+Das macht seine Argumente für Missverständnisse und Verstellungen
+anfällig, was unserer Meinung nach aus dem offenen Brief, in dem seine
+Absetzung gefordert wird, klar hervorgeht. Es muss in der Interprätation
```
Comment (louis77) [inline index_de.md:34]: "Bei der Interpretation seiner Worte sollte daher berücksichtigt werden"

File: index_de.md [hunk 6]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+seiner Worte daher berücksichtigt werden, dass eine diplomatische
+Ausdrucksweise weder sein Metier ist, noch unabdingbar für seinen
```
Comment (louis77) [inline index_de.md:36]: "Metier" = "Fachbereich".
So if he engages in "language purism", diplomacy could indeed be his "Metier". 
But, IMO the original text says more like "it is not his goal to ... pursue diplomacy".
Proposal: "seine undiplomatische Ausdrucksweise keine Voraussetzungs ist für seinen aktivistischen Erfolg"?

File: index_de.md [hunk 7]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+aktivistischen Erfolg.
+
+Unabhängig davon, sind die eigentlichen Meinungen Stallmans, für die er
+in diesem Fall anbeprangert wird, nicht relevant hinsichtlich seiner
```
Comment (louis77) [inline index_de.md:40]: typo: "an*g*eprangert"
weglassen: "in diesem Fall" because this attack goes on for long now

File: index_de.md [hunk 8]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+Fähigkeit, eine Gemeinschaft wie die FSF zu führen. 
+Darüber hinaus hat er denselben Anspruch auf eine eigene Meinung, 
+wie jeder andere auch. Mitglieder und Unterstützer müssen nicht zwingend
+mit allen seinen Ansichten einverstanden sein, und sollten sein Recht auf
```
Comment (louis77) [inline index_de.md:44]: remove ","

File: index_de.md [hunk 9]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+Gedanken- und Meinungsfreiheit respektieren.
 
 **An die FSF:**
 
-Das Absetzen von RMS schadet dem Image der FSF und wäre ein 
-erheblicher Schlag gegen die Dynamik der Freien Software-Bewegung.
-Wir bitten euch, euer Handeln sorgfältig zu überprüfen. Denn was ihr
-entscheiden werdet, wird auf die Zukunft der Softwarebranche ernsthafte
-Auswirkungen haben.
+Das Absetzen von RMS schadet dem ansehen der FSF und wäre ein
```
Comment (louis77) [inline index_de.md:49]: typo: *A*nsehen

File: index_de.md [hunk 10]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
 
 **An die FSF:**
 
-Das Absetzen von RMS schadet dem Image der FSF und wäre ein 
-erheblicher Schlag gegen die Dynamik der Freien Software-Bewegung.
-Wir bitten euch, euer Handeln sorgfältig zu überprüfen. Denn was ihr
-entscheiden werdet, wird auf die Zukunft der Softwarebranche ernsthafte
-Auswirkungen haben.
+Das Absetzen von RMS schadet dem ansehen der FSF und wäre ein 
+erheblicher Schlag gegen der Freiheit in der Freien Software-Bewegung.
```
Comment (louis77) [inline index_de.md:50]: "momentum" != "Freiheit"
What is wrong with "Dynamik"?

File: index_de.md [hunk 11]
```diff
@@ -4,52 +4,52 @@ layout: signed
...
+Wir bitten euch, euer Handeln sorgfältig zu bedenken — denn eure Entscheidung
+könnte die Zukunft der Freien Softwarebranche ernsthaft beeinträchtigen.
```
Comment (louis77) [inline index_de.md:52]: That is too much of a deviation from the original text. 
"will have a serious impact" cannot be translated with "könnte ... beeinträchtigen".
In the original text there is also no mention of "Freie Softwarebranche".

**Review by louis77 — CHANGES_REQUESTED** [review]
Hier mein Input

**Comment by imachug** [comment]
Thanks a lot!

**Pull request merged**